lib/rentlinx/models/base.rb

```diff
@@ -1,11 +1,13 @@
 module Rentlinx
   class Base
     def initialize(attrs)
+      @processor = AttributeProcessor.new(attrs.dup)
```

Comment:
Should _base_ directly require the files it depends on? I know it gets that functionality for free from the main module, but then this code is dependent on that being correct